services/core-web/.eslintrc.js

```diff
@@ -40,5 +40,6 @@ module.exports = {
     "jsx-a11y/label-has-for": 0, // deprecated rule
     "import/no-cycle": 0,
     "no-param-reassign": 0,
+    "no-return-assign": 0,
```

Comment:
I feel like we can do without this one, personally. Line 12 in documentGeneration.js was the cause in this case.